src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -3191,14 +3191,89 @@
 short-summary: Manage private endpoints.
 """
 
+helps['network private-endpoint create'] = """
+type: command
+short-summary: Create a private endpoint.
+examples:
+  - name: Create a private endpoint.
+    text: az network private-endpoint create -g MyResourceGroup -n MyPE --vnet-name MyVnetName --subnet MySubnet --private-connection-resource-id MyPLSId --connection-name tttt -l centralus
```

Comment:
just wondering about this one:

--private-connection-resource-id MyPLSId

shouldn't that be resource id?

Reply:
yes. I will change this example.